main.py

```diff
@@ -225,40 +227,201 @@ async def usage(ctx):
+  rounds_idx = 0
+
+  reactions = [ ["🔴", "⚫"] ,  ["⬆️", "⬇️"],  ["📥", "📤"],  ["♥️", "♦️", "♣️", "♠️"]]
+  cards_pulled = []
+
+  color_match = {"🔴": ["hearts", "diamonds"], "⚫": ["clubs", "spades"]}
+  
+  face_values = {"J": 11, "Q": 12, "K": 13, "A": 14} # not sure how this works for Ace yet, probably need to do both ranges...?
+
+  suit_match = {"hearts": "♥️", "diamonds": "♦️", "clubs": "♣️", "spades": "♠️"}
+
+  def check(reaction, user): # pass this with each round index.
+    return user == ctx.author
+  
+  while winnings >= 0 and rounds_idx < len(rounds):
+    curr_card = deck.pull_card()
+
+    curr_msg = "<@!{0}>\n".format(ctx.author.id)
+    curr_msg += rounds[rounds_idx]
+    if rounds_idx > 0:
+      curr_msg += "\nPrevious cards: "
+      for card in cards_pulled:
+        curr_msg += "{0}, ".format(card.get_card())
+      curr_msg = curr_msg[:-2]
+      curr_msg += "\nYou can stop betting here by reacting with :octagonal_sign: to take your winnings, or try and bet for more!"
+      curr_msg += "\nYou have {0} won so far!".format(winnings)
+    await ctx.send(curr_msg)
+
+    try:
+      reaction, user = await bot.wait_for("reaction_add", timeout=30.0, check=check)
+      
+      if str(reaction.emoji) == "🛑" and rounds_idx > 0:
+        await ctx.send("You stopped betting. You won {0} Boga Bucks!".format(winnings))
+        break
+      
+      round_win = False
+      if str(reaction.emoji) in reactions[rounds_idx]:
+        # continue betting. 
+        if rounds_idx == 0: # red or black. 
+          if curr_card.get_suit() in color_match[str(reaction.emoji)]:
+            round_win = True
+          else:
+            round_win = False
+            
+        # second round.
+        elif rounds_idx == 1:
+          card_1, card_2 = cards_pulled[0].get_value(), curr_card.get_value()
+          
+          if card_1 in face_values:
+            card_1 = face_values[card_1]
+          if card_2 in face_values:
+            card_2 = face_values[card_2]
+          
+          card_1 = int(card_1)
+          card_2 = int(card_2)
+
+          if card_1 < card_2 and str(reaction.emoji) == "⬆️":
+            round_win = True
+          elif card_1 > card_2 and str(reaction.emoji) == "⬇️":
+            round_win = True
+          else:
+            round_win = False
+       
+        elif rounds_idx == 2:
+          # third round. 
+          card_1, card_2 = cards_pulled[0].get_value(), cards_pulled[1].get_value()
+          card_3 = curr_card.get_value()
+          
+          if card_1 in face_values:
+            card_1 = face_values[card_1]
+          if card_2 in face_values:
+            card_2 = face_values[card_2]
+          if card_3 in face_values:
+            card_3 = face_values[card_3]
+          
+          card_1 = int(card_1)
+          card_2 = int(card_2)
+          card_3 = int(card_3)
+
+          card_range = range(card_1, card_2)
+          
+          if card_3 in card_range and str(reaction.emoji) == "📥":

```

Comment:
```suggestion
          if str(reaction.emoji) == "📥" and card_3 in card_range:
```

I think it's a little clearer this way, and should be a little faster cause of short circuiting

Reply:
Swapped to suggested way.